merlin-driver/src/main/java/gov/nasa/jpl/aerie/merlin/driver/MissionModelLoader.java

```diff
@@ -68,23 +67,16 @@ public static MerlinPlugin loadMissionModelProvider(final Path path, final Strin
         final var className = getImplementingClassName(path, name, version);
 
         // Construct a ClassLoader with access to classes in the mission model location.
-        final var parentClassLoader = Thread.currentThread().getContextClassLoader();
-        final var classLoader = new URLClassLoader(new URL[] {missionModelPathToUrl(path)}, parentClassLoader);
+        final var classLoader = new URLClassLoader(new URL[] {missionModelPathToUrl(path)});
 
         try {
             final var factoryClass$ = classLoader.loadClass(className);
             if (!MerlinPlugin.class.isAssignableFrom(factoryClass$)) {
                 throw new MissionModelLoadException(path, name, version);
             }
 
-            // SAFETY: We checked above that MissionModelFactory is assignable from this type.
-            @SuppressWarnings("unchecked")
-            final var factoryClass = (Class<? extends MerlinPlugin>) factoryClass$;
-
-            return factoryClass.getConstructor().newInstance();
-        } catch (final ClassNotFoundException | NoSuchMethodException | InstantiationException
-            | IllegalAccessException | InvocationTargetException ex)
-        {
+            return (MerlinPlugin) factoryClass$.getConstructor().newInstance();
```

Comment:
Figure no SAFETY comment necessary, because there's the assignable check above?

Reply:
That's right. Would you prefer I add one anyway for clarity?

Reply:
No need, I think it's quite clear as is.